api/api.go

```diff
@@ -26,7 +26,7 @@ const (
 	APPLICATION_JSON = "application/json"
 )
 
-var cliUserAgentExp = regexp.MustCompile("^(?i)(curl|wget|fetch\\slibfetch)\\/.*$")
+var cliUserAgentExp = regexp.MustCompile(`^(?i)((curl|wget|fetch\slibfetch)\/.*|Go\s1\.1\spackage\shttp)$`)
```

Comment:
Wouldn't this match only Go 1.1? Something like `Go\s.*\spackage\shttp` might be better?